src/yardstick/tool/grype.py

```diff
@@ -154,9 +154,34 @@ def _install_from_git(
 
         return Grype(path=path, version_detail=description, **kwargs)
 
+    @classmethod
+    def _install_from_path(
+        cls,
+        path: Optional[str],
+        src_path: str,
+        **kwargs,
+    ) -> "Grype":
+        # get the description and head ref from the repo
+        src_repo_path = os.path.abspath(src_path)
+        logging.debug(f"installing grype from path={src_repo_path!r}")
+        logging.debug(f"installing grype to path={path!r}")
+        if not path:
+            path = tempfile.mkdtemp()
+            atexit.register(shutil.rmtree, path)
+        dest_path = os.path.join(path.replace("path:", ""), "local_install")
+        os.makedirs(dest_path, exist_ok=True)
+        cls._run_go_build(
+            abspath=os.path.abspath(dest_path),
+            description=f"local_build:{src_path}",
```

Comment:
the description is typically used as the version specified in the ld-flags as well as the version on the python `Grype` object (in the constructor the `version_details`) like here https://github.com/anchore/yardstick/pull/111/files#diff-eba0a362cbd33d93d1a6ed803e109b7da6fde23ce94f3118f577ad8140ce8b3aL155 . This way the version makes it into the scan configuration.

The goal is to try and make the description distinguishable from other grype versions. This is why the git describe is used in other approaches (maybe with `--dirty` makes sense too here).